src/components/BoxForm/DropZone/DropZone.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --dropzone-color: #FAFAFA;
+}
+.dropzone-zone {
+  text-align: center;
+  padding: 55px;
+  border-radius: 5px;
+  border-width: 2px;
+  border-style: dashed;
+  border-color: #b9b9b9;
+  background-color: var(--dropzone-color);
+  color: #b9b9b9;
+}
+
+.dropzone-accept {
+  border-color: #00e676;
+}
+
+.dropzone-reject {
+  border-color: #ff1744;
+}
+
+.dropzone-text {
+  color: black;
+}
+.dropzone-support-text {
+  font-size: 0.8rem;
+}
+
+.fileSelector {
+  border: none;
+  background-color: transparent;
+  color: black;
+  /* font-weight: 700; */
+  text-decoration: underline;
+  font-size: 1rem;
+  cursor: pointer;
+}
+
+.files-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.file-item button {
```

Comment:
Can we change this so the button and span tags have their own class names? It is better to specify the class name for each component you want to style rather than keep it general to all button or span tags, just in case we choose to add another button/span tag later on.